### DISCUSSION TITLE
Proper way to store api jwt token

### DISCUSSION THREAD

**Discussion by vv1n3k** [nextauthjs/next-auth#745] [Help]
**Your question**
I wonder how to pass JWT token from API to Next Auth session. Is it safe solution to store the info in NextAuth token provided with encryption flag set to true? 

**What are you trying to do**
I'm using a credential provider and can fetch the user from the backend. I would like to store my JWT API token and be able to use it in further client side requests or authorized server side API calls requested.

**Feedback**
I would be great to prepare some additional tutorials describe how to iteract with custom API. Callback section need better documentation. BTW your solution seems nice. 

* [x] Found the documentation helpful
* [x] Found documentation but was incomplete
* [ ] Could not find relevant documentation
* [x] Found the example project helpful
* [ ] Did not find the example project helpful

**Comment by kenshinman** [comment]
Hello @vv1n3k have you found any solution to this yet?
